src/shared/modules/connections/connectionsDuck.js

```diff
@@ -134,6 +141,11 @@ const mergeConnectionHelper = (state, connection) => {
   )
 }
 
+// Local vars
+let memoryUsername = ''
```

Comment:
@oskarhane I am a bit concerned about having those as local variable, although the current lifecycle of duck files are Ok with that. We may have them in redux store but not synced with local storage.